native/core/src/execution/datafusion/expressions/structs.rs

```diff
@@ -125,3 +125,103 @@ impl PartialEq<dyn Any> for CreateNamedStruct {
             .unwrap_or(false)
     }
 }
+
+#[derive(Debug, Hash)]
+pub struct GetStructField {
```

Comment:
I think this can move to the `spark-expr` crate?

Reply:
Yeah I thought about that, but seemed to make sense to co-locate with CreateNamedStruct. Is it ok to move both in this PR, or do you want me to just move this new one?

Reply:
I agree that CreateNamedStruct should also be in spark-expr, so I think it is fine to move it as part of this PR

Reply:
Migrated both